packages/listbox/src/ListboxMixin.js

```diff
@@ -714,6 +689,16 @@ const ListboxMixinImplementation = superclass =>
       this._listboxNode.focus();
     }
 
+    /**
+     * @protected
```

Comment:
TODO: add comment on why we have `_scrollIntoView ` method (to allow Subclassres to adjust the animation: like non smooth behavior, different timing etc.)